scripts/us_bls/cpi/cpi_u_1913_2020.mcf

```diff
@@ -0,0 +1,7 @@
+Node: dcid:ConsumerPriceIndex_ConsumerGoodsAndServices_UrbanConsumer_BLSSeasonallyUnadjusted
```

Comment:
(Noticed that this is duplicate of c_cpi_u, which is ok, but) do we need StatVars to be repeated in order for your automation pipeline to work? Or could we do with one file referred to by both c_cpi and c_cpi_u?

Reply:
We can do with one file. I will remove this file.

Reply:
removed